print/locales/de-CH-scout.json

```diff
@@ -1 +1,22 @@
-{}
+{
+  "contentNode": {
+    "storycontext": {
+      "add": "Roten Faden hinzufügen",
```

Comment:
`add` key is obsolete
And this complete files should probably be empty. I think there was a mistake between source and destination.